### ISSUE TITLE
Can't install/load MSGFplus package on Windows

### ISSUE THREAD

**Issue by JorgeVindelAlfageme** [thomasp85/MSGFplus#20]
Hi:
If I install the MSGFplus library using this line of code:

> BiocManager::install("MSGFplus")

And, after that, if I try loading it, what I get is this warning message:

> library(MSGFplus)
Error: package or namespace load failed for ‘MSGFplus’:
 .onLoad failed in loadNamespace() for 'MSGFplus', details:
  call: system2(.javaExecutable(), "-version", stderr = TRUE, stdout = TRUE)
  error: 'CreateProcess' failed to run 'C:\PROGRA~1\Java\JDK18~1.0_3 -version'

On the other hand, if I try installing this package after removing it using this code line:

> devtools::install_github('thomasp85/MSGFplus')

This message appears:

> \* installing *source* package 'MSGFplus' ...
** using staged installation
** R
Error in .install_package_code_files(".", instdir) : 
files in 'Collate' field missing from 'C:/Users/jvalf/AppData/Local/Temp/Rtmp6T1b6Q/R.INSTALL164c4e697167/MSGFplus/R':
  msgfParGUI.R
ERROR: unable to collate and parse R files for package 'MSGFplus'
\* removing 'C:/Users/jvalf/Documents/R/win-library/4.0/MSGFplus'

I just need it to work in order to install "proteomics" library. What do I need to do, exactly? Thanks in advance.

**Comment by JorgeVindelAlfageme** [comment]
Update: I forked the repo from @thomasp85 account and after that I uploaded the previously deleted msgfParGUI.R file which I recovered from thomasp85/MSGFplus/R site into the forked repo. Once I did that, I just ran the code line:

>devtools::install_github('JorgeVindelAlfageme/MSGFplus') 

I restarted my RStudio session and it seems like it correctly installed. I don't know if this will give me more issues in the future, so I still kindly wait for your response @thomasp85 in order to give a more detailed explanation about what has happened during the installation process.

**Comment by thomasp85** [comment]
MSGFplus is no longer being supported and has been removed from Bioconductor, partly because Java on Windows has become a mess. So whatever you do that works is absolutely the best way forward🙂

**Comment by JorgeVindelAlfageme** [comment]
Ok. Thank you for your answer, Thomas.
Regards.

**Comment by JorgeVindelAlfageme** [comment]
I would like to ask you something else, even though this issue is not completely related to my new question: I decided to install your package because I needed to use another package which is involved in working with proteomics data. The thing is I'm a novel research working in a proteomics lab, and my boss thought these packages would help us processing the data they produce in the lab, but if this package is not in Bioconductor anymore, this means that maybe the state of the art in computational biology for proteomics is a different set of tools. Considering all this, I would like to ask you if you know what the most used bioinformatic tools for proteomics are nowadays. In any case, thank you for your attention.

**Comment by thomasp85** [comment]
I honestly couldn't say. I haven't been involved in bioinformatics, let alone proteomics, in many years. My guess is that other packages in bioconductor has risen since then

**Comment by JorgeVindelAlfageme** [comment]
It's alright. I'll see what I can find. Thank you for your responses, Thomas.
Regards.

**Comment by JorgeVindelAlfageme** [comment]
I've reopened this issue just to include one final message: The problem when I was trying to install the package "RforProteomics" was that some other packages, such as MSGFplus, were given me installing issues. Today, I've had more problems related to other package and in order to solve it I've reinstalled the last version of R (which currently is 4.1.2). After that, I could install all the packages just by using the commands that are shown in Bioconductor without any other problem. So all the issues I had could be solved just by updating R. I wanted to post this last comment if any other person who see this can use this recommendation.
